### PR TITLE
Custom dropdown

### PR DESCRIPTION
**Pull request by tobycm**
hell yeah!

### CODE REVIEW

File: src/App.tsx
Comment (pdt1806) [inline src/App.tsx:1]: bruh

Comment (tobycm) [reply src/App.tsx:1]: it's `to` right?

**Review by pdt1806 — APPROVED** [review]
image

File: src/App.tsx
Comment (pdt1806) [reply src/App.tsx:1]: it is

**Comment by tobycm** [comment]
imagine someone trying to contribute see this pr

**Comment by pdt1806** [comment]
> imagine someone trying to contribute see this pr

image

**Pull request merged**